src/scrape_up/devCommunity/articles.py

```diff
@@ -0,0 +1,110 @@
+            "{'data': '{\n'
+         '  "articles": [\n'
+         '    {\n'
+         '      "Title": "With GitHub apparently down right now...",\n'       
+         '      "Author": "Ben Halpern",\n'
+         '      "Date": "Jun 29",\n'
+         '      "Tags": "#watercooler\\n#github",\n'
+         '      "Reactions": "22\\u00a0reactions",\n'
+         '      "Comments": "Comments\\n\\n              6\\u00a0comments",\n'
+         '      "Read Time": "2 min read"\n'
+         '    },\n'
+         '    {\n'
+         '      "Title": "Sloan\'s Inbox: What is the Secret to a Truly Great '
+         'Portfolio?",\n'
+         '      "Author": "Sloan the DEV Moderator",\n'
+         '      "Date": "Jun 29",\n'
+         '      "Tags": '
+         '"#discuss\\n#beginners\\n#career\\n#careerdevelopment",\n'
+         '      "Reactions": "10\\u00a0reactions",\n'
+         '      "Comments": "Comments\\n\\n              1\\u00a0comment",\n'  
+         '      "Read Time": "1 min read"\n'
+         '    },\n'
+         '    {\n'
+         '      "Title": "Sticky sessions with Apache APISIX",\n'
+         '      "Author": "Nicolas Frankel",\n'
+         '      "Date": "Jun 29",\n'
+         '      "Tags": '
+         '"#stickysessions\\n#sessionaffinity\\n#apacheapisix\\n#devops",\n'
+         '      "Reactions": "44\\u00a0reactions",\n'
+         '      "Comments": "Comments\\nAdd\\u00a0Comment",\n'
+         '      "Read Time": "4 min read"\n'
+         '    }
+         '  ]\n'
+         '}',
+ 		'message': 'Successfully fetched all articles.'}
+        
+        ```
+		"""
+		page = self.__scrape_page()
+		articles_list = {"articles": []}
+		try:
+			articles = page.find_all( class_="substories")
+			for x in articles:
+				title = x.find_all(class_="crayons-story__title")        
+				author = x.find_all('a', class_="crayons-story__secondary fw-medium m:hidden")
+				date = x.find_all('a', class_="crayons-story__tertiary fs-xs")       
+				tags = x.find_all(class_="crayons-story__tags")
+				reactions = x.find_all('span', class_="aggregate_reactions_counter")        
+				comments = x.find_all('a',class_="crayons-btn crayons-btn--s crayons-btn--ghost crayons-btn--icon-left flex items-center")        
+				readTime = x.find_all(class_="crayons-story__save")
+	
+			for ti,a,d,ta,r,c,re, in zip(title, author, date, tags, reactions, comments, readTime):
+				articles_list["articles"].append({
+				"Title": ti.text.strip(),
+            			"Author": a.text.strip(),
+            			"Date": d.text.strip(),
+            			"Tags": ta.text.strip(),
+            			"Reactions": r.text.strip(),
+            			"Comments": c.text.strip(),
+            			"Read Time": re.text.strip(),
+            			})
+			json_data = json.dumps(articles_list, indent=2)
+			return {"data": json_data, "message": "Successfully fetched all articles."}
+		except:
+			message = "An Error Occurred!"
+			return message
+
+dev = DevCommunity()
+pprint(dev.all_articles())
+
```

Comment:
The code looks good.

Do not create a separate file for `articles`, `tags`, and `user`. Instead, add them as a method to the `DevCommunity` class.
So that we can call the methods after creating one class only.

Reply:
Okay I will make the adjustments and update the PR.